core/CMakeLists.txt

```diff
@@ -13,21 +13,21 @@ else()
 endif()
 
 # We need tinyxml2 for the camera definition parsing.
-if(IOS OR APPLE)
+if(APPLE)
     # We install the tinyxml2 library manually for macOS and iOS.
     # The install is commented out as long as we build statically.
```

Comment:
Actually this comment is obsolete now, I guess it should be removed.

Also note that we never built tinyxml2 as a static library, I believe (and we'll need that for iOS). It requires `BUILD_STATIC_LIBS` to be set, which I guess we don't want to set in the whole scope of `core/CMakeLists.txt`. I am planning to work a bit more on the build system, but after I get the prototype running on iOS :-).

Reply:
Thanks, removed the comment.